test/src/main/java/org/apache/accumulo/test/functional/MemoryFreeingIterator.java

```diff
@@ -28,23 +28,39 @@
 import org.apache.accumulo.core.iterators.IteratorEnvironment;
 import org.apache.accumulo.core.iterators.SortedKeyValueIterator;
 import org.apache.accumulo.core.iterators.WrappingIterator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
 public class MemoryFreeingIterator extends WrappingIterator {
 
-  @Override
-  public void init(SortedKeyValueIterator<Key,Value> source, Map<String,String> options,
-      IteratorEnvironment env) throws IOException {
-    super.init(source, options, env);
+  private static final Logger LOG = LoggerFactory.getLogger(MemoryFreeingIterator.class);
+
+  @SuppressFBWarnings(value = "DM_GC", justification = "gc is okay for test")
+  public MemoryFreeingIterator() {
+    super();
+    LOG.info("Freeing consumed memory");
```

Comment:
Well, it's attempting to anyway... there's no guarantee here.

```suggestion
    LOG.info("Attempting to free consumed memory");
```

Reply:
Addressed in 55265e8199